app/src/services/team.service.js

```diff
@@ -10,7 +10,7 @@ class TeamService {
       const baseURL = config.get("teamsAPI.url");
```

Comment:
does this contain the new v3 team endpoint URL?

Reply:
Yes